cluster-up/cluster/kind/common.sh

```diff
@@ -141,6 +141,28 @@ function _fix_node_labels() {
     done
 }
 
+function _get_cri_bridge_mtu() {
+  docker network inspect bridge  | grep mtu | awk '{print $2}' | sed "s/\"//g"
```

Comment:
Please try `docker network inspect -f '{{index .Options "com.docker.network.driver.mtu"}}' bridge`

Reply:
I couldn't find how to read the inner map inside Options with `--format`, very nice! 
It works I will change this